pkg/proxy/proxy.go

```diff
@@ -83,48 +99,107 @@ func getProxy(namespace string, name string, serviceCAfile string, cacheManager
 		TLSHandshakeTimeout: tlsHandshakeTimeout,
 	}
 
-	targetURL := fmt.Sprintf("http://tempo-%s-query-frontend.%s:3200", name, namespace)
-	proxyURL, err := url.Parse(targetURL)
+	var targetURL string
+	switch tempo.Kind {
+	case api.TempoStackKind:
+		if len(tempo.Tenants) > 0 {
+			service := DNSName(fmt.Sprintf("tempo-%s-gateway", tempo.Name))
+			targetURL = fmt.Sprintf("https://%s.%s.svc:8080/api/traces/v1/%s/tempo", service, tempo.Namespace, url.PathEscape(tenant))
```

Comment:
are this ports also static or can they be configured by customers?

Reply:
The are static.